src/features/controls.ts

```diff
@@ -265,16 +279,23 @@ export const ControlsMixin = (ModelViewerElement:
           this[$controls].setOrbit(theta, phi, radius as number);
         }
 
+        [$updateCameraTarget]() {
+          let target = deserializeVector3(this.cameraTarget);
+
+          if (target == null) {  // this happens in the default 'auto' case
```

Comment:
Let's deserialize to `'auto'` in this case. The code will be more self-explanatory, and it will line up with what we do for deserializing sphericals.

Reply:
Relatedly, it seems likely that I would want to be able to set `'auto'` per-axis of the vector. For example, maybe I want the target to be automatically selected for me along the `y` axis, while panning the camera along the `x` axis.

Reply:
Yeah, I was on the fence about that already. Sounds good.